docs/content/guides/developer/sui-101/send-txn.mdx

```diff
@@ -1,6 +1,197 @@
 ---
-title: Sending Transactions
-draft: true
+title: Transactions
```

Comment:
```suggestion
title: Creating Transactions
```
The problem here is that we already have a page titled Transactions (although it is in the Concepts section where this one is in Guides)...https://docs.sui.io/concepts/transactions. This is my suggestion.